build-common/NHibernate.props

```diff
@@ -13,7 +13,7 @@
     <FileVersion Condition="'$(VersionSuffix)' != '' AND '$(BuildNumber)' != ''">$(VersionPrefix).$(BuildNumber)</FileVersion>
     <FileVersion Condition="'$(FileVersion)' == ''">$(VersionPrefix).0</FileVersion>
 
-    <NhAppTargetFrameworks Condition ="$(NhAppTargetFrameworks) == ''">net461;netcoreapp2.0</NhAppTargetFrameworks>
+    <NhAppTargetFrameworks Condition ="$(NhAppTargetFrameworks) == ''">net461;netcoreapp2.1</NhAppTargetFrameworks>
```

Comment:
In order to run test in VS 2022, the NUnit test adapter has to be updated to its fourth version at least. But this version does no more support target netcoreapp2.0, and requires at least netcoreapp2.1.

May we have better to directly target net6.0 for our tests?

Reply:
I think 2.1 should be enough.